### PR TITLE
Update Wasp CLI bash completion list

### LINKED ISSUE #754: Update Wasp CLI bash completion list
Wasp's bash completion support is implemented here: https://github.com/wasp-lang/wasp/blob/1995ccdb9d9dd6d17d880e298b9b1305ca016994/waspc/cli/src/Wasp/Cli/Command/BashCompletion.hs .

In the meantime we added some new commands to Wasp, and so it got somewhat outdated.
We should update it to correctly reflect current commands that Wasp CLI has.

### PR DESCRIPTION
**Pull request by ussgarci**
# Description

This PR updates the Wasp CLI bash completion list to reflect the current commands in the Wasp CLI.

Fixes #754

## Type of change

Please select the option(s) that is more relevant.

- [ ] Code cleanup
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

**Comment by h4r1337** [comment]
LGTM @Martinsos

File: waspc/cli/src/Wasp/Cli/Command/BashCompletion.hs [hunk 1]
```diff
@@ -26,7 +26,7 @@ bashCompletion = do
     ["db", cmdPrefix] -> listMatchingCommands cmdPrefix dbSubCommands
     _ -> liftIO . putStrLn $ ""
   where
-    commands = ["new", "version", "start", "db", "clean", "build", "telemetry", "deps"]
+    commands = ["new", "version", "waspls", "start", "db", "clean", "build", "telemetry", "deps", "info"]
```
Comment (Martinsos) [inline waspc/cli/src/Wasp/Cli/Command/BashCompletion.hs:29]: We also have `completion` command, you can also add that one! And `completion:generate`. These are not mentioned but they exist, check them here, @h4r1337 just added the docs for them: https://github.com/wasp-lang/wasp/pull/766/files .

**Comment by Martinsos** [comment]
@h4r1337 I agree, therefore I created https://github.com/wasp-lang/wasp/issues/773 !

File: waspc/cli/src/Wasp/Cli/Command/BashCompletion.hs (hunk 1 above)
Comment (ussgarci) [reply waspc/cli/src/Wasp/Cli/Command/BashCompletion.hs]: Added!  Thank you.  :bow:

**Comment by Martinsos** [comment]
Great @ussgarci , merged!!!

**Pull request merged**